### PR TITLE
I Love Music Presence

### PR DESCRIPTION
**Pull request by MauriceX24**
Preview

### CODE REVIEW

File: ilovemusic/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    "en": "The best Web-Radio of Germany - The best Music for everyone!",
+    "de": "Das beste Internet-Radio in Deutschland - Die beste Musik für jeden!"
+  },
+  "url": "www.ilovemusic.de",
+  "version": "1.0",
+  "logo": "https://i.imgur.com/jO86Aqk.png",
+  "thumbnail": "https://i.imgur.com/LFMmAHc.png",
+  "color": "#F7C516",
+  "tags": ["music", "internet radio"],
+"category": "music"}
```
Comment (Bas950) [inline ilovemusic/dist/metadata.json]: please fix that ending
and tags cannot include spaces use -

File: ilovemusic/dist/presence.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
+//import { write } from "fs";
+
+console.warn("Starting I Love Music Presence");
+console.log(document);
```
Comment (Bas950) [inline ilovemusic/dist/presence.js]: you should remove this

Comment (MauriceX24) [reply ilovemusic/dist/presence.js]: Done! ^^

File: ilovemusic/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{
+  "author": {
+    "name": "MauriceX",
+    "id": "268768664411439115"
+  },
+  "service": "I Love Music",
```
[the remark this answers is not in the document]
Comment (MauriceX24) [reply ilovemusic/dist/metadata.json]: Done!

File: I Love Music/dist/metadata.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    "en": "The best Web-Radio of Germany - The best Music for everyone!",
+    "de": "Das beste Internet-Radio in Deutschland - Die beste Musik für jeden!"
+  },
+  "url": "www.ilovemusic.de",
+  "version": "1.0",
+  "logo": "https://i.imgur.com/jO86Aqk.png",
+  "thumbnail": "https://i.imgur.com/LFMmAHc.png",
+  "color": "#F7C516",
+  "tags": ["music","internet-radio","radio"],
+  "category": "music"}
```
Comment (Bas950) [inline I Love Music/dist/metadata.json]: please put the } on its own line, as its more readable that way xD

File: I Love Music/dist/presence.js [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+var playback;
+var browsingStamp = Math.floor(Date.now() / 1000);
+if (lastPlaybackState != playback) {
+    lastPlaybackState = playback;
+    browsingStamp = Math.floor(Date.now() / 1000);
+}
+
+presence.on("UpdateData", () => __awaiter(this, void 0, void 0, function* () {
+    var a = presenceData = '', presenceData = {
+        largeImageKey: "logo"
```
Comment (Bas950) [inline I Love Music/dist/presence.js]: var a = presenceData = '',  can be removed lmao

Comment (MauriceX24) [reply I Love Music/dist/presence.js]: `presence.on("UpdateData", () => __awaiter(this, void 0, void 0, function* () {
    var a = presenceData = {
        largeImageKey: "logo"  
    };`
If I remove this I get an error

File: I Love Music/dist/metadata.json (hunk 4 above)
Comment (MauriceX24) [reply I Love Music/dist/metadata.json]: Alright :D

File: I Love Music/dist/presence.js (hunk 5 above)
Comment (Bas950) [reply I Love Music/dist/presence.js]: https://docs.premid.app/en/dev/presence#getting-started-1

Comment (MauriceX24) [reply I Love Music/dist/presence.js]: Thank you x) 
I removed this `a = `

**Pull request merged**